src/models/user.cpp

```diff
@@ -258,14 +172,15 @@ void User::setWorkspacePath(const QString &workspacePath)
 
 bool User::operator ==(const User &u)
 {
-    return u.getAddress() == getAddress() && u.getCity() == getCity() && u.getCompany() == getCompany() &&
-            u.getEmail() == getEmail() && u.getFirstname() == getFirstname() && u.getLastname() == getLastname()&&
-            u.getMobilePhone() == getMobilePhone() && u.getNoSiret() == getNoSiret() && u.getPhone() == getPhone()&&
-            u.getPostalCode() == getPostalCode() && u.getTitle() == getTitle();
+    return  People::operator == (u)
```

Comment:
Tu peux pas faire this == u ici ? Ou les caster en People pour que ça fonctionne ?